<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IFOCUSODC-PC47" ignored="0" name="AuthApitest" tests="12" failures="5" timestamp="29 Mar 2018 06:33:05 GMT" time="46.101" errors="0">
  <testcase name="authentication_positive" time="3.375" classname="AP1_Prod.Authentication_Prod"/>
  <testcase name="getauthkey_positive" time="4.526" classname="AP1_Prod.GetAuthKey_Prod"/>
  <testcase name="SendMail" time="4.908" classname="AP1_Prod.SendEmail"/>
  <testcase name="SetPassword" time="6.345" classname="AP1_Prod.SetPassword_Prod">
    <failure type="java.lang.AssertionError" message="isPosted value is not as expected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: isPosted value is not as expected expected [false] but found [true]at AP1_Prod.SetPassword_Prod.SetPassword(SetPassword_Prod.java:63)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- SetPassword -->
  <testcase name="changePassword" time="4.262" classname="AP1_Prod.ChangePassword_Prod">
    <failure type="java.lang.AssertionError" message="isPosted value is not as expected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: isPosted value is not as expected expected [false] but found [true]at AP1_Prod.ChangePassword_Prod.changePassword(ChangePassword_Prod.java:70)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- changePassword -->
  <testcase name="getprofile_positive" time="5.166" classname="AP1_Prod.GetProfileUser_Prod"/>
  <testcase name="SetUsername" time="4.271" classname="AP1_Prod.SetUsername_Prod">
    <failure type="java.lang.AssertionError" message="isPosted value is not as expected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: isPosted value is not as expected expected [false] but found [true]at AP1_Prod.SetUsername_Prod.SetUsername(SetUsername_Prod.java:70)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- SetUsername -->
  <testcase name="checkemail_V1" time="4.212" classname="AP1_Prod.CheckMail_Prod">
    <failure type="java.lang.AssertionError" message="isExist value is not as expected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: isExist value is not as expected expected [false] but found [true]at AP1_Prod.CheckMail_Prod.checkemail_V1(CheckMail_Prod.java:65)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- checkemail_V1 -->
  <testcase name="kuserlogin_positive" time="0.492" classname="AP1_Prod.KUserLogin_Prod"/>
  <testcase name="createapi" time="3.301" classname="AP1_Prod.Create_Prod"/>
  <testcase name="kuserreg_positive" time="0.979" classname="AP1_Prod.KUserReg_Prod"/>
  <testcase name="edituser" time="4.226" classname="AP1_Prod.EditUser_Prod">
    <failure type="java.lang.AssertionError" message="isExist value is not as expected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: isExist value is not as expected expected [false] but found [true]at AP1_Prod.EditUser_Prod.edituser(EditUser_Prod.java:54)... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- edituser -->
</testsuite> <!-- AuthApitest -->
